lib/ssl/test/dtls_api_SUITE.erl

```diff
@@ -529,14 +529,14 @@ client_restarts_multiple_acceptors(Config) ->
                 {ok, IntSocket} = gen_statem:call(Pid, {downgrade, self()}),
                 {{Address,CPort},UDPSocket}=IntSocket,
                 ?CT_LOG("Info: ~p~n", [inet:info(UDPSocket)]),
-                {ok, #config{transport_info = CbInfo, connection_cb = ConnectionCb,
+                {ok, #config{transport_info = CbInfo,
                              ssl = SslOpts0}} =
-                    ssl:handle_options(ClientOpts, client, Address),
+                    ssl_config:handle_options(ClientOpts, client, Address),
                 SslOpts = {SslOpts0, #socket_options{}, undefined},
                 ct:sleep(250),
                 ?CT_LOG("Client second connect: ~p ~p~n", [Socket, CbInfo]),
-                {ok, NewSocket} = ssl_gen_statem:connect(ConnectionCb, Address, CPort, IntSocket,
-                                                         SslOpts, self(), CbInfo, infinity),
+                {ok, NewSocket} = dtls_gen_connection:start_fsm(client, Address, CPort, IntSocket,
```

Comment:
what was wrong with this testcase?

Reply:
Well it was modified to match my other PR, as I hade one branch before but I will change it back and it will be changed by the other PR.